docs/plugins/discovery-azure.asciidoc

```diff
@@ -56,9 +56,11 @@ discovery:
 .Binding the network host
 ==============================================
 
+WARNING: The keystore file must be placed in a directory accessible by elasticsearch like the `config` directory.
```

Comment:
Indeed. That's exactly what was blocking me in my branch. It makes sense to me to have that in config.